README.md

```diff
@@ -1,25 +1,25 @@
+# ![recheck logo](https://user-images.githubusercontent.com/1871610/41766965-b69d46a2-7608-11e8-97b4-c6b0f047d455.png) for web apps
+
 [![Build Status](https://travis-ci.com/retest/recheck-web.svg?branch=master)](https://travis-ci.com/retest/recheck-web)
 [![license](https://img.shields.io/badge/license-AGPL-brightgreen.svg)](https://github.com/retest/recheck-web/blob/master/LICENSE)
 [![PRs welcome](https://img.shields.io/badge/PRs-welcome-ff69b4.svg)](https://github.com/retest/recheck-web/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
 [![code with hearth by retest](https://img.shields.io/badge/%3C%2F%3E%20with%20%E2%99%A5%20by-retest-C1D82F.svg)](https://github.com/retest)
 
-# recheck-web
-
-recheck for web apps. Replace manual asserts and check everything at once.
+[recheck](https://github.com/retest/recheck) for web apps. Replace manual asserts and check everything at once.
 
 ## Features
 
-* Easy creation and maintenance of checks in web tests.
+* Easy creation and maintenance of checks for web.
 * Semantic comparison of contents.
 * Easily ignore volatile elements, attributes or sections.
 * One-click maintenance to update tests with intended changes.
 * No unexpected changes go unnoticed.
-* Operates on top of Selenium.
 * The Git for your GUI.
+* Operates on top of Selenium.
```

Comment:
According to "Marketing Speak", the strongest should be last. Strongest here is "The Git for the GUI". This is what should be remembered ... not "Operates on top of Selenium".